examples/rotation.c

```diff
@@ -47,13 +47,17 @@ static void handle_output_frame(struct output_state *output, struct timespec *ts
 	wlr_renderer_clear(sample->renderer, &(float[]){0.25f, 0.25f, 0.25f, 1});
 
 	float matrix[16];
+	float alpha = 1.0;
 	for (int y = -128 + (int)odata->y_offs; y < height; y += 128) {
 		for (int x = -128 + (int)odata->x_offs; x < width; x += 128) {
 			wlr_texture_get_matrix(sample->cat_texture, &matrix,
 				&wlr_output->transform_matrix, x, y);
 			wlr_render_with_matrix(sample->renderer,
-					sample->cat_texture, &matrix);
+				sample->cat_texture, &matrix, alpha);
 		}
+		alpha -= 0.1;
+		if (alpha < 0.0)
+			alpha = 1.0;
```

Comment:
style - use braces here

Reply:
Also, this seems like a weird place to test alpha. Can we add a command to rootston to set the alpha of a window?